src/components/wallet/UserTransactionsTable.tsx

```diff
@@ -12,63 +12,79 @@ type Props = {
   transactions: UserTransactionType[]
 }
 
-const TransactionsTable: React.FC<Props> = ({ transactions }): JSX.Element => {
-  const intl = useIntl()
+type Column = 'type' | 'address' | 'to' | 'amount' | 'coin'
 
-  const address = 'tbnb1vxutrxadm0utajduxfr6wd9kqfalv0dg2wnx5y'
+const getColumnsRenderers = (address: string): Record<Column, (value: any, tx: UserTransactionType) => JSX.Element> => {
```

Comment:
Try to avoid `any`.

Reply:
@veado i totally agree with but here is no any chance to that. Internal antd's table typings  provide only this.
![image](https://user-images.githubusercontent.com/67793194/86753702-f5aaa100-c048-11ea-9ca3-7819d40fc06c.png)
we also can not use `unknown` here 'casue in this case there will not be a possibility to type `coin` renderer